tests/memory_maps/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+define_shared_lib(
+  LIBNAME 2
+  SRCS 2.c
+  PKEY 2
+)
+
+define_shared_lib(
+  LIBNAME 3
+  SRCS 3.c
+  PKEY 3
+)
+
+define_shared_lib(
+  LIBNAME 4
+  SRCS 4.c
+  PKEY 4
+)
+
+define_test(
+  SRCS main.c
+  PKEY 1
+  NEEDS_LD_WRAP
+  CRITERION_TEST
+  LIBS 2 3 4
+)
```

Comment:
@fw-immunant, do I need `WITHOUT_SANDBOX` here, like `permissive_mode/CMakeLists.txt` has?  `fopen("/proc/self/maps", "r")` is failing with `ia2-sandbox`.

Reply:
Okay now it's still hanging indefinitely in CI even with `WITHOUT_SANDBOX`, which fixed it locally.